@kiva/kv-components/tests/unit/specs/components/KvPagination.spec.js

```diff
@@ -0,0 +1,132 @@
+import { render } from '@testing-library/vue';
+import userEvent from '@testing-library/user-event';
+import KvPagination from '../../../../vue/KvPagination.vue';
+
+global.scrollTo = jest.fn();
+
+describe('KvPagination', () => {
+	afterEach(jest.clearAllMocks);
+
+	it('should render arrows disabled by default', async () => {
+		const { getByLabelText, emitted } = render(KvPagination, { props: { limit: 10, total: 0 } });
+
+		await userEvent.click(getByLabelText('Previous page'));
```

Comment:
Last I heard `user.setup` was the "new" approach to click events in `testing-library`. Has the recommendation changed?

Reply:
Yeah, that's right. The problem is that the project is using `13.2.1` . I tried upgrading it to `14.2.0` version, but the upgrade caused some weird errors in KvTextInput and KvTabs tests

Reply:
I we will have to upgrade the lib which comes with some refactoring of other tests to move to the new approach.

Reply:
Sounds good, thanks.